.github/workflows/ci.yaml

```diff
@@ -315,10 +315,43 @@ jobs:
           path: ./logs/
           retention-days: 15
 
-  push-gpu-operator-bundle-image:
+  release-latest-gpu-operator-and-validator-image:
     if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }} # Runs only if the event is a push to the main branch
     needs: [e2e-tests-containerd, e2e-tests-nvidiadriver]
-    runs-on: ubuntu-latest
+    runs-on: linux-amd64-cpu4
+    steps:
+      - uses: actions/checkout@v4
+        name: Check out code
+      - name: set-up regctl
+        run: |
+          export REGCTL_VERSION=v0.7.1
+          mkdir -p bin
+          curl -sSLo bin/regctl https://github.com/regclient/regclient/releases/download/${REGCTL_VERSION}/regctl-linux-amd64
+          chmod a+x bin/regctl
+          echo "$(pwd)/bin" >> $GITHUB_PATH
+      - name: Calculate test vars
+        id: vars
+        run: |
+          COMMIT_SHORT_SHA=${GITHUB_SHA:0:8}
+          echo "COMMIT_SHORT_SHA=${GITHUB_SHA:0:8}" >> $GITHUB_ENV
+          LOWERCASE_REPO_OWNER=$(echo "${GITHUB_REPOSITORY_OWNER}" | awk '{print tolower($0)}')
+          echo "LOWERCASE_REPO_OWNER=$(echo "${GITHUB_REPOSITORY_OWNER}" | awk '{print tolower($0)}')" >> $GITHUB_ENV
+          echo "OPERATOR_VERSION=${COMMIT_SHORT_SHA}" >> $GITHUB_ENV
+          echo "OPERATOR_IMAGE=ghcr.io/${LOWERCASE_REPO_OWNER}/gpu-operator" >> $GITHUB_ENV
+          echo "VALIDATOR_VERSION=${COMMIT_SHORT_SHA}" >> $GITHUB_ENV
+          echo "VALIDATOR_IMAGE=ghcr.io/${LOWERCASE_REPO_OWNER}/gpu-operator/gpu-operator-validator"  >> $GITHUB_ENV
+      - name: Retag gpu-operator and gpu-operator-validator
+        env:
+          ADDITIONAL_IMAGE_TAG: ${{ github.ref_name == 'main' && 'main-latest' || '' }}
```

Comment:
I don't think we need this variable. We can just hardcode `main-latest` since it's already implied from the CI stage name
```suggestion
```